includes/paytabs_payment_methods.php

```diff
@@ -497,6 +499,61 @@ public function process_refund($order_id, $amount = null, $reason = '')
         return $success;
     }
 
+    public  function process_capture($order_id)
+    {
+        global $woocommerce;
+
+        $order = wc_get_order($order_id);
+        $amount = $order->get_total();
+        $transaction_id = $order->get_transaction_id();
+
+        if (!$transaction_id) {
+            return false;
+        }
+        
+
+        // PT
+        $currency = $order->get_currency();
+
+        //$order_id = $order->get_id();
+        $reason = 'Admin request';
+
+        $payment_id = $this->getPaymentMethod($order);
+        if ($payment_id == $this->id)
+        {
+            $pt_capHolder = new PaytabsFollowupHolder();
+            $pt_capHolder
+                ->set02Transaction(PaytabsEnum::TRAN_TYPE_CAPTURE, PaytabsEnum::TRAN_CLASS_ECOM)
+                ->set03Cart($order_id, $currency, $amount, $reason)
+                ->set30TransactionInfo($transaction_id)
+                ->set99PluginInfo('WooCommerce', $woocommerce->version, PAYTABS_PAYPAGE_VERSION);
+
+            $values = $pt_capHolder->pt_build();
+
+            $_paytabsApi = PaytabsApi::getInstance($this->paytabs_endpoint, $this->merchant_id, $this->merchant_key);
+            $capRes = $_paytabsApi->request_followup($values);
+
+            $success = $capRes->success;
+            $message = $capRes->message;
+            $pending_success = $capRes->pending_success;
+
+            $order->add_order_note('Capture status: ' . $message, true);
+
+            if ($success) {
+                //$order->update_status('Completed', __('Payment captured: ', 'PayTabs'));
+            } else if ($pending_success) {

```

Comment:
All failed captures should change the Order status, not only the pending_success